test/trap/ok/divInt32.wasm-run.ok

```diff
@@ -3,6 +3,7 @@ Error: failed to run main module `_out/divInt32.wasm`
 Caused by:
     0: failed to invoke command default
     1: error while executing at wasm backtrace:
-         0: init
-         1: _start
-    2: wasm trap: integer overflow
+         0: div<Int32>
```

Comment:
Difference in inlining?

Reply:
Probably introduced here https://github.com/dfinity/motoko/pull/4225/files#r1598296884

Reply:
Yes, this is the reason.